.unit-test/proxy-test.js

```diff
@@ -1,25 +1,49 @@
-//
-//(function () {
-//    'use strict';
-//    describe('Test GameBoardService', function () {
-//        var constants,
-//            theState,
-//            dataHandler;
-//        beforeEach(function(){
-//            module('Services.MyModule');
-//            inject(function($injector, $http, $q,  _DataHandle_){
-//                theState = $http;
-//                constants = $injector.get('Proxy');
-//                dataHandler = _DataHandle_;
-//            });
-//        });
-//        it('Checks if the newGame function updates the playerTypes on a new game', function(){
-//            var thedata = constants.newGame('human','human');
-//            should(constants.thedata).to.equal('newGame', ['human','human']);
-//        });
-//
-//        afterEach( function(){
-//
-//        });
-//    });
-//}());
+
+(function () {
+    'use strict';
+    describe('Test Proxy', function () {
+        var httpBackend,
+            returnedPromise,
+            proxyName;
+        beforeEach(function(){
+            module('Services.MyModule');
+            inject(function( $httpBackend, $q, _Proxy_  ){
+                httpBackend = $httpBackend;
+                returnedPromise = $q;
+                proxyName = _Proxy_;
+
+            });
+        });
+        it('Checks if the newGame function in the proxy service returns the correct values', function(){
+            var theResponse = {'outcome':'Continue','gameboard':'000000000','winner':0};
+            httpBackend.expectPOST("http://eutaveg-01.tombola.emea:35000/api/v1.0/newgame" , {'player1':"human", 'player2':"human"})
+                .respond(theResponse);
+            var returnedPromise = proxyName.newGame("human", "human");
+            var result;
+            returnedPromise.then(function(response){
```

Comment:
Put var declarations at the top: 
the "returnedPromise.then ..." statement interrupts this.